rero_ils/modules/locations/permissions.py

```diff
@@ -34,8 +34,9 @@ def list(cls, user, record=None):
         :param record: Record to check.
         :return: True is action can be done.
         """
-        # List locations allowed only for staff members (lib, sys_lib)
-        return current_patron and current_patron.is_librarian
+        if not current_patron:
```

Comment:
In one line we can write `return True if current_patron else False`

Reply:
done